tests/fast-integration/skr-svcat-migration-test/test-helpers.js

```diff
@@ -180,10 +185,10 @@ async function cleanupInstanceBinding(creds, svcatPlatform, btpOperatorInstance,
     try {
         args = [`unbind`, btpOperatorInstance, btpOperatorBinding, `-f`, `--mode=sync`];
         await execa(`smctl`, args);
-        // let {stdout} = await execa(`smctl`, args);
-        // if (stdout !== "Service Binding successfully deleted.") {
-        //     errors = errors.concat([`failed "smctl ${args.join(' ')}": ${stdout}`])
-        // }
+        let {stdout} = await execa(`smctl`, args);
```

Comment:
`smctl` exit code is 0 even on failure for `unbind`, we need to check the stdout to make sure it suceeded